nautobot/circuits/models.py

```diff
@@ -43,7 +43,7 @@ class Provider(PrimaryModel):
         verbose_name="ASN",
         help_text="32-bit autonomous system number",
     )
-    account = models.CharField(max_length=30, blank=True, verbose_name="Account number")
+    account = models.CharField(max_length=50, blank=True, verbose_name="Account number")
```

Comment:
My vote would actually be to go big on this and increase it to 100 just to avoid having to potentially change this again in the future. We can discuss tomorrow.

Reply:
Done